test/e2e/azure_suite_test.go

```diff
@@ -257,3 +265,22 @@ func writeLogs(mgmt *kind.Cluster, namespace, deploymentName, logDir string) err
 	}
 	return nil
 }
+
+func ensureCAPZArtifactsDeleted(input *ControlPlaneClusterInput) {
+	input.SetDefaults()
+
+	mgmtClient, err := input.Management.GetClient()
+	gomega.Expect(err).NotTo(gomega.HaveOccurred(), "stack: %+v", err)
+
+	By("Deleting cluster")
+	ctx := context.Background()
+	gomega.Expect(mgmtClient.Delete(ctx, input.Cluster)).NotTo(gomega.HaveOccurred())
+
+	gomega.Eventually(func() []clusterv1.Cluster {
+		clusters := clusterv1.ClusterList{}
+		c, err := input.Management.GetClient()
+		gomega.Expect(err).NotTo(gomega.HaveOccurred())
+		gomega.Expect(c.List(ctx, &clusters)).NotTo(gomega.HaveOccurred())
+		return clusters.Items
+	}, input.DeleteTimeout, 20*time.Second).Should(gomega.HaveLen(0))
```

Comment:
s/gomega.// here to use the existing `.` import for gomega

Reply:
and this

Reply:
(And same with all the other `gomega` uses in this func)

Reply:
done